### PR TITLE
Add Average time by month report

### PR DESCRIPTION
**Pull request by jestradadeveloper**
# Description
Add route and action in employees controller for get a view of the average time of check in check outs in the current month
What did you implemented/Why did you implemented this:
 - Add a route for avg time of check in/out by the current month of the employees
 - Add an action to the employees_controller for render a view of avg time report
 - View for render every employee avg time of working time in the current month
 - Refactor of component Checkin code for set the error messages accessing to the message in the right way

# Testing

rails s

rspec spec/models/employee_spec.rb:54

 Example:
  - I've added a test for get_total_avg_time on the Employee model

### CODE REVIEW

**Review by JuanequeX — APPROVED** [review]
LGTM

File: app/javascript/components/CheckIn/CheckIn.jsx [hunk 1]
```diff
@@ -21,22 +22,23 @@ const Checkin = ({ authenticity_token, checkIn, checkOut }) => {
...
   }
+
   const checkOutEmployee = secret => {
     axios.post('/check-out', {
       private_number: secret
     }).then((res) => {
-      const { message } = res.data[0];
-      setSuccessMessage(message)
+      const { message } = res.data;
+      setSuccessMessage(message[0]);
```
Comment (OswaldoPineda) [inline app/javascript/components/CheckIn/CheckIn.jsx:31]: you can use `message.first` to get the first element

**Review by OswaldoPineda — APPROVED** [review]
nice

File: app/javascript/components/CheckIn/CheckIn.jsx (hunk 1 above)
Comment (jestradadeveloper) [reply app/javascript/components/CheckIn/CheckIn.jsx:31]: Right we can do that, we add it in a next PR. Thanks

**Review by CUBOFIG — APPROVED** [review]
LGTM!

**Pull request merged**